src/display/xfa_layer.js

```diff
@@ -30,13 +81,20 @@ class XfaLayer {
         Object.assign(html.style, value);
       }
     }
+
+    // Set the value after the others to be sure overwrite
+    // any other values.
+    if ("fieldId" in attributes && storage) {
```

Comment:
Nit: It's probably a tiny bit cheaper to re-order this as `if (storage && attributes.fieldId !== undefined) {` instead.